### ISSUE TITLE
Tests: WOPI validator

### ISSUE THREAD

**Issue by wkloucek** [owncloud/ocis#5111]
https://github.com/owncloud/ocis/pull/5089 introduced the WOPI validator in CI for:
- personal spaces

We need to run the WOPI validator also for:
- public links
- shares
- ...

Places where we should run the WOPI validator:
- [x] oCIS
- [ ] REVA
- [x] ~WOPI server~ collaboration service

**Comment by dragotin** [comment]
I think the validator should run in the WOPI Server repo. @tbsbdr @micbar can we please put that in the backlog? It is an important part of the product.

**Comment by micbar** [comment]
We have already a ticket in the repo https://github.com/cs3org/wopiserver/issues/57

**Comment by micbar** [comment]
I proposed this already, was somehow dropped from their roadmap.

**Comment by dragotin** [comment]
Yes, we can not expect that CERN is hardening our oCIS product for which we need the WOPI. I think the task should be in our backlog.

**Comment by wkloucek** [comment]
> Places where we should run the WOPI validator:
> ...
> - [ ] WOPI server

Could be considered as a checkmark now, since oCIS 6 / 7 has the collaboration service now instead of the WOPI server.

**Comment by wkloucek** [comment]
@micbar today found something interesting in https://github.com/owncloud/ocis/pull/10462#issuecomment-2455722817:

> While digging into the upstream repo, i found this https://github.com/microsoft/wopi-validator-core/issues/22#issuecomment-1407225548 which means the upstream status of the WOPI Validator is unclear.

**Comment by micbar** [comment]
@wkloucek @individual-it 

That is correct. Until now this is just an announcement. Things didn’t change so far.

Please Note: if you use ms365 in a production env, every user can just create a *.wopitest file and open in in ms365 cloud. This will run the same validator in the WebUI.
